### ISSUE TITLE
[L] Divergent links (same name, different URLs)

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/czi-scientific-python-mgmt#88]
Divergent links (term I invented) - two or more links that have the **same name** but go to **different URLs**. 

These are particularly bad for people who use speech command software.

```[tasklist]
### Tasks
- [ ] Find divergent links in PyData Theme docs
- [ ] Fix divergent links in PyData Theme docs
- [ ] Add section in accessibility playbook about link writing - link to it from the PST docs
```

**Comment by gabalafou** [comment]
Tania says that PST does not have the ability to run checks on the content for theme adopters.

Bonus task: Could be useful to have a GitHub action that checks for divergent links.

**Comment by gabalafou** [comment]
I marked this as a large task because it's quite a bit of effort to go through every link on every page of the docs. 

Might be better to create some code or GitHub action to do this, but that is also considerable time.
